src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -323,7 +341,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       const extendedPositionWithKeyboard = Math.max(
         0,
         animatedContainerHeight.value -
-          (animatedSheetHeight.value + keyboardHeightInContainer)
+        (animatedSheetHeight.value + keyboardHeightInContainer)
```

Comment:
I think this was correctly indented before - can you revert this change?

Reply:
It looks correctly indented locally, so I'm not completely sure why it shows it like this.